jerry-core/parser/regexp/re-compiler.c

```diff
@@ -559,7 +562,7 @@ re_compile_bytecode (const re_compiled_code_t **out_bytecode_p, /**< [out] point
   re_ctx.flags = flags;
   re_ctx.highest_backref = 0;
   re_ctx.num_of_non_captures = 0;
-
```

Comment:
Is the removal of this empty line separator intentional?

Reply:
No, it wasn't intentional. I added a line separator after the `REGEXP_RECURSION_COUNTER_INIT` macro.